cypress/integration/home_page_spec.js

```diff
@@ -2,12 +2,14 @@ describe('Home page user flow', () => {
     beforeEach(() => {
         cy.onLoad()
     });
-
+// this first test is skipet because we don't have all the JSX elements yet
     it.skip('Should contain the Naviagtion bar with all elements ', () => {
         cy.get()
           .should('be.visible')
           .get()
           .contains()
+
+        //   we need to add all the elements that will contain in the home page - later also test for the animations
```

Comment:
I like that you added the tests already for what will be in place in the future. Smart thinking! 🧐